frontend/js/src/utils/utils.tsx

```diff
@@ -131,6 +131,61 @@ const searchForYoutubeTrack = async (
   return null;
 };
 
+const searchForSoundcloudTrack = async (
+  soundcloudToken?: string,
```

Comment:
This should probably not be allowed to be undefined

Reply:
Fixed